proto/localstate.proto

```diff
@@ -10,100 +10,100 @@ import "localstatetypes.proto";
 service LocalState {
     // Get only the raw data from a datastore UTXO that has been mined into chain
     rpc GetData(GetDataRequest) returns (GetDataResponse) {
-      option(google.api.http) = {
+      option (google.api.http) = {
           post: "/v1/get-data"
           body: "*"
         };
     }
     // Get a list of UTXOs that sum to at least a minimum of some value where each
     // UTXO has a common owner
     rpc GetValueForOwner(GetValueRequest) returns (GetValueResponse) {
-      option(google.api.http) = {
+      option (google.api.http) = {
```

Comment:
Fine with change but please document linter/formatter that added spaces so that we do not continue reverting changes due to auto-formatting.

Reply:
This was a manual change that I detailed in the PR description. Basically to not break syntax highlighting in IDEs.